ocr/pogoWindows.py

```diff
@@ -667,7 +671,11 @@ def __internal_get_inventory_text(self, filename, identifier, x1, x2, y1, y2):
         gray = cv2.resize(gray, dim, interpolation=cv2.INTER_AREA)
         cv2.imwrite(temp_path_item, gray)
         with Image.open(temp_path_item) as im:
```

Comment:
corrupt/partial file transfers might cause Image.open to fail here, I suggest moving the try/catch block outside this `with` block

Reply:
actually the file reference I meant is probably read on line 659 (first line of method)